src/Utils/IslandoraBaggerUtils.php

```diff
@@ -80,8 +80,6 @@ public function getIslandoraBaggerConfig($path) {
       return FALSE;
     }
 
-    // Use a static cache to avoid reading the config file multiple times.
-    static $settings;
     $settings = Yaml::parseFile($path);
```

Comment:
Why do you want to remove the static cache here?

Reply:
For the record, discussed elsewhere: "had to do with parsing a changed yaml file before saving."